spec/display-buffer-spec.coffee

```diff
@@ -128,6 +128,12 @@ describe "DisplayBuffer", ->
           expect(displayBuffer.tokenizedLineForScreenRow(4).tokens[0].isSoftWrapIndentation).toBeTruthy()
           expect(displayBuffer.tokenizedLineForScreenRow(4).tokens[1].isSoftWrapIndentation).toBeTruthy()
 
+        it "correctly tokenizes hanging indentation spaces", ->
```

Comment:
Please note that I wrote only a test that verifies the correct tokenization of `hangingIndentationSpaces`, as positioning was already thoroughly tested by other specs. I believe that adding similar ones for these hanging spaces was redundant and not worth the maintenance.